### DISCUSSION TITLE
[Projects Beta] Allow back-linking for issues referenced in another repo

### DISCUSSION THREAD

**Discussion by dmulter** [community/community#14434] [Projects and Issues]
I create epic issues with task lists enumerating specific children issues. When I provide a `#111` reference, a backlink in the child issues is created to the parent epic. The task list item with that issue also auto-completes when the child issue is closed. This is as expected for Projects Beta. Thank you!

The problem is if I reference a child issue in another repo. An absolute URL or a short version `ORG/REPO#111` both link correctly from parent to child issue, but do NOT do the backlink or task list auto-complete when closed.

FYI, we often have issues that span multiple repos when implementing backend and frontend work. I would think this is a very common use case. Also note that I am not looking for complex epic support, just basic task list issue linking that works fully for an issue in any repo.

**Comment by azenMatt** [comment]
Hi @dmulter 

I have been able to recreate that this doesn't auto-complete the checklist when closed, however I do get the backlink. Do you mind sharing some more information on this - perhaps you could blur our some screenshots and show us where it is missing?

**Reply by dmulter** [reply]
Steps to repro along with expected/actual results. Note that converting task list items into issues creates them in the same repo and then everything works fine. For context, I'm using the following technique for doing epics.

* Create an issue in repo A numbered `1`
* Create an issue in repo A numbered `2`
* Create an issue in repo B numbered `3`
* Add a task list item `#2` to reference issue `2` in the same repo
  * Auto-complete works for selecting the issue
  * The issue # is replaced by the title text of the issue
  * Hovering over the issue shows more detail included a `Tracked In` link to this parent issue `1`
  * Also note that you can use `ORG/REPO#2`, the full URL, or search for an issue when adding the task list item
* Click on issue `2`
  * `Tracked In` link references back to parent issue `1`
* Add a task list item `#3` to reference issues `3` in the other repo
  * Unexpected: The task list item shows just as the strong `#3`
  * Unexpected: Note that search does NOT work
* Add a task list item `ORG/REPO#3`
  * The task list item is replaced by the title text of the issue
  * Unexpected: Hovering over the issues shows more detail, but does NOT include the `Tracked In` link
  * Note that you can use the full URL for issue `3` and get the same results
  * Click on issue `3` from the task list
  * Unexpected: Note that issue `3` does NOT have a `Tracked In` link back to parent issue `1`

Interpretation:
* Cross referencing issues across repos does not fully work.
* I've verified that Project values are not producing the issue, the different repos are.
* The `Tracked In` linking is very important to expose epic relationships as developers will primarily work in child issues and the epic gives critical context for the detailed work in each issue.
* Using `#1` or `ORG/REPO#1` style issue references is MUCH easier than full URLs.
* Lack of search across repos makes linking tougher.

**Reply by azenMatt** [reply]
Thanks @dmulter - I can recreate this 

We are looking into this and also have several updates coming soon to improve the tracked in pill and general experience here that will help

**Reply by moovinzoo** [reply]
Is there any update?
I think tracked badge is still not working on child issue of tasklist in different repo.
